src/file_input/MIDI_file_class.py

```diff
@@ -92,6 +96,28 @@ def calc_time_delay(self):
         if self.curr_tempo is None or self.curr_time_signature is None:
             return None
         
+    
+    def is_correct_hand(self, note_num):
+        """
+        NOTE: The hand_to_play side refers to the side the PIANO is going to play.
+        The player would, presumably, be playing the opposite.
+
+        
+        If the given note is not on the hand_to_play side return false
+        """
+        #print(f'Hand to play: {self.hand_to_play}')
+        if self.hand_to_play == "right":    # The piano will play right hand
+            if note_num >= 60:
+                return True
+        elif self.hand_to_play == "left":   # The piano will play left hand
+            if note_num <= 60:
+                return True
+        elif self.hand_to_play == "":       # The piano will play all
+            return True
+        elif self.hand_to_play == "neither": # The piano will play nothing
```

Comment:
Hadn't thought about this, but yeah hat could totally be a case. add this to the enum too then